#
# Copyright (c) 2019 Intel Coporation.
#
# This work is licensed under the terms of the MIT license.
# For a copy, see <https://opensource.org/licenses/MIT>.
#

Header header

float32 velocity
float32 acceleration
geometry_msgs/Quaternion orientation
CarlaEgoVehicleControl control
